plugins/autoloader/prism-autoloader.js

```diff
@@ -25,18 +25,37 @@
 	}
 	var config = Prism.plugins.autoloader = {
 		languages_path: languages_path,
-		use_minified: true
+		use_minified: true,
+		sri: {
```

Comment:
Instead of an `sri` key specifically, can we modify the autoloader so it's hookable so we can implement the sri? That would be more generic and make the autoloader less likely to change in the future. May even be less code, if some of the current logic is moved into the sri plugin.